src/tools/rustfmt/tests/target/pin_sugar.rs

```diff
@@ -7,3 +7,12 @@ fn f(x: &pin const i32) {}
 fn g<'a>(x: &'a pin const i32) {}
 fn h<'a>(x: &'a pin mut i32) {}
 fn i(x: &pin mut i32) {}
+
+struct Foo;
+
+impl Foo {
+    fn f(&self) {}
+    fn g<'a>(&'a self) {}
+    fn h<'a>(&'a mut self) {}
+    fn i(&mut self) {}
+}
```

Comment:
Is this correct? `pin const` is removed from the `f` and `g` methods, and `pin` is dropped from the `h` and `i` methods.

Reply:
Oh, thanks for correcting me. I didn't notice the codes have been changed by `--bless` when running tests.